app/index.html

```diff
@@ -24,6 +24,8 @@
   <script type="text/javascript" src="js/livebus/livebus.factory.js"></script>
 
   <script type="text/javascript" src="js/shared/shared.module.js"></script>
+  <script type="text/javascript" src="js/shared/ucBackground/ucBackground.module.js"></script>
+  <script type="text/javascript" src="js/shared/ucBackground/ucBackground.directive.js"></script>
```

Comment:
I'm saying that the filenames should **not** use the directive prefix. Neither should the module (package) name. Only the directive name should use the prefix.

All the other files in this project don't have `uc` suffix, so these should not be an exception.
In other words the filenames should be:
`background.image.module.js`
`background.image.directive.js`